_pages/sponsors.md

```diff
@@ -1,13 +1,33 @@
 ---
 layout: page
-title: Sponsors
-subtitle: Sponsors
 permalink: /sponsors/
 ---
 
 <div class="pretty-links">
 
-We're currently looking for sponsors!
+# Sponsors
+
+<br>
+
+![](/assets/img/sponsors/MEF.png)
+
+The [Mathematics Endowment Fund](https://uwaterloo.ca/math-endowment-fund/about) (MEF) is an income-generating fund that finances projects for the academic betterment of mathematics undergraduate students at the University of Waterloo. Members of the StarCon organizing committee have used MEF funding for previous projects on the Women in CS committee and for attending conferences. We're very grateful for their support. If you are a Waterloo math student and not in town for StarCon, we recommend reaching out to them for travel funding!
+
+[![](/assets/img/sponsors/stripe.png)](test.com)
+
+[Stripe](https://stripe.com/) is a software platform for starting and running internet businesses. Hundreds of thousands of businesses—including Lyft, Kickstarter, Salesforce, Shopify, Facebook, Slack, and UNICEF—rely on Stripe’s software tools to accept payments, expand globally, and create new revenue streams. Stripe has been at the forefront of expanding internet commerce, powering new business models, and supporting the latest platforms, from marketplaces to mobile commerce sites.
```

Comment:
we're giving them anything they want in the top level (I think?) and they gave us the most money, which is why I decided to put them first. Not super set on that though if anyone has strong opinions